fiatconnect-api.md

```diff
@@ -1496,7 +1516,9 @@ the risk of creating a duplicate transaction. Idempotency keys MUST be implement
 
 #### 3.4.4.1. `POST /transfer/in`
 
-The `POST /transfer/in` endpoint is used to initiate a new transfer in from fiat to crypto.
+The `POST /transfer/in` endpoint is used to initiate a new transfer in from fiat to crypto. Since the transfer of fiat funds from a user to a provider
```

Comment:
This is more a comment than anything, but the /transfer/in could be irrelevant in a user-initiated transfer world. With some CICO partners, the transfer details will always remain constant for each user, meaning that the user could simply send money to the transfer details and the onramp should follow.

To not change the specs too much, i think we can keep this, but just pointing out that if the transfer details of Fiat are always the same, technically the partner does not need to call this endpoint to `initiate` the onramp.